### DISCUSSION TITLE
Mark without padding

### DISCUSSION THREAD

**Discussion by eridurancosta** [iSoron/uhabits#1436] [General Discussion]
Hi! I have a twice a week habit. I already checked two days ando the test of the days I get a check mark without padding, only with the border. What does it mean?
![Screenshot_20220722-091904-475](https://user-images.githubusercontent.com/109803328/180437840-b0e7c5ab-230e-47d5-8cfc-05ddef432e1c.png)

**Comment by hiqua** [comment]
Check https://github.com/iSoron/uhabits/discussions/689,  "What are unfilled checkmarks and why do they keep appearing?"
